TFLite request to support bulit-in split op for int64 type

**System information**
- OS Platform and Distribution MacOS
- TensorFlow installed from: recent nightly build, after issue https://github.com/tensorflow/tensorflow/issues/50445
- TensorFlow version (or github SHA if from source): master HEAD



**Standalone code to reproduce the issue** 


```
def test_split_export():

    import tensorflow as tf

    @tf.function
    def test_fn(inp: tf.Tensor) -> tf.Tensor:
        return tf.concat(tf.split(inp, 3, axis=1), axis=1)

    concrete_f = test_fn.get_concrete_function(tf.TensorSpec((None, 3), tf.int64))
    converter = tf.lite.TFLiteConverter.from_concrete_functions([concrete_f])

    tflite_model = converter.convert()
    interpreter = tf.lite.Interpreter(model_content=tflite_model)
    interpreter.allocate_tensors()
```

after fix #50445, running this test issues the following converter error: 

```
E         tensorflow.lite.python.convert_phase.ConverterError: /Users/dmlyubim/opt/anaconda3/envs/p38/lib/python3.8/site-packages/tensorflow/python/ops/array_ops.py:2129:0: error: 'tf.Split' op is neither a custom op nor a flex op
```

We think it is a fairly easily fixable gap in the tflite builtin op coverage. 

The supporting arguments are as follows: 

* The main argument is that int64 type is supported in tflite2 built-in set; in fact, the concat op as in following test, works no problem:

```
def test_concat_export():
    import tensorflow as tf

    @tf.function
    def test_fn(*inp: tf.Tensor) -> tf.Tensor:
        return tf.concat(inp, axis=1)

    concrete_f = test_fn.get_concrete_function(
        *([tf.TensorSpec((None, 3), tf.int64)] * 3)
    )
    converter = tf.lite.TFLiteConverter.from_concrete_functions([concrete_f])

    tflite_model = converter.convert()
    interpreter = tf.lite.Interpreter(model_content=tflite_model)
    interpreter.allocate_tensors()
```

Split is a complementary (reverse) operation of the tensor concat op, we think there's no reason to support one but not another. 

* It should be a rather easy fix (split is already supported for int32 etc., so adding new type should be much easier than if we had to create a whole new op).

* In the age when models are transplantable, licensable, and/or expensive to train, any work-around solutions assuming model re-graphing or/and re-training to work around int64 issue are non-solutions. We have to deal with the topology and models we already have a license to evaluate. 

* Using TF_SELECT_OPS runitme introduces some problems with our requirements (mostly size, but also build system). This is a smaller and perhaps weaker support argument but it is still a big trouble for us mostly on requirement grounds (we are not targeting Android platform, or at least not exclusively; but we do target a C++ environment, not python). 

* Also, this op was identified as the only missing op precluding us from transition from TFLite1 to TFLite2. That's the only operator that really does not work. Everything else seems to be covered by built-ins for the widest range of models we investigated.

* Oddly enough, our tflite conversions (and evaluations) for the same models worked in TFLite1. Probably due to another bug or mysterious context of the op use in the models we observed; a similar int64 input and split as in the example, does not work with TFLite1.15 test either for me. However, it mysteriously works for different contexts of int64 split use in the models we tried to convert with TFLite 1.15 but not TFLite2.

--- a/tensorflow/lite/kernels/split.cc
+++ b/tensorflow/lite/kernels/split.cc
@@ -87,7 +87,7 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
   TF_LITE_ENSURE(context,
                  input_type == kTfLiteFloat32 || input_type == kTfLiteUInt8 ||
                      input_type == kTfLiteInt8 || input_type == kTfLiteInt16 ||
-                     input_type == kTfLiteInt32);
+                     input_type == kTfLiteInt32 || input_type == kTfLiteInt64);
   for (int i = 0; i < NumOutputs(node); ++i) {
     TfLiteTensor* tensor;
     TF_LITE_ENSURE_OK(context, GetOutputSafe(context, node, i, &tensor));
@@ -158,6 +158,10 @@ TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
       TF_LITE_SPLIT(int32_t);
       break;
     }
+    case kTfLiteInt64: {
+      TF_LITE_SPLIT(int64_t);
+      break;
+    }     
     default:
       TF_LITE_KERNEL_LOG(context, "Type %s currently not supported.",
                          TfLiteTypeGetName(op_context.input->type));

